serviceconnector/test/.eslintrc.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+    "extends": [
+        "@microsoft/eslint-config-azuretools",
+        "@microsoft/eslint-config-azuretools/test"
+    ],
+    "rules": {
+        "no-restricted-imports": "off"
```

Comment:
Curious, where/what are we importing that we needed to turn this rule off?

Reply:
I was mostly basing this off the other packages since they all turn this rule off. Based on some more investigation it seems like the base rule needs to be turned off since it can return incorrect errors. For more info here's the [link](https://typescript-eslint.io/rules/no-restricted-imports/).